### ISSUE TITLE
[Compatibility Issue] Lanterns Bug with WorldEdit's Copy and Paste

### ISSUE THREAD

**Issue by wdh7987** [thedarkcolour/Future-MC#228]
When copying and pasting a set of blocks that includes a hanging lantern and the block it was hanging from, the chain that would normally symbolize a lantern that is hanging is not pasted with everything else. Probably the NBT data of the lantern is not being saved with the copy.

**Comment by thedarkcolour** [comment]
It's metadata? Does copy + paste work with other modded blocks?

**Comment by wdh7987** [comment]
Yeah, it's the only block added by this mod that has issues from what I've seen.

**Comment by thedarkcolour** [comment]
Where are you pasting the block? Are you trying to make a floating hanging lantern?

**Comment by wdh7987** [comment]
I am pasting the block attached to a fence-constructed lamp post. An example of what I'm copying and pasting is in the attached image.
![2021-03-26_00 42 44](https://user-images.githubusercontent.com/70856103/112579300-4a0ad200-8dcc-11eb-8ec0-75f0dabe2d4e.png)
